jre_emul/android/platform/libcore/harmony-tests/src/test/java/org/apache/harmony/tests/java/util/zip/DeflaterInputStreamTest.java

```diff
@@ -24,11 +24,16 @@
 import java.util.zip.DataFormatException;
 import java.util.zip.Deflater;
 import java.util.zip.DeflaterInputStream;
-
-import junit.framework.TestCase;
 import libcore.io.Streams;
+//import libcore.junit.junit3.TestCaseWithRules;
```

Comment:
Could you follow tho conventions when we comment out lines due to J2ObjC-specific needs? Check other parts of jre_emul where we use block comments with a line "J2ObjC removed: [rationale]" as the first line. We can also note a TODO item for this.

Reply:
added comments